applications/callflow/src/module/cf_resources.erl

```diff
@@ -372,7 +372,7 @@ handle_channel_destroy(_, JObj) ->
 get_privacy_prefs(Call) ->
     case use_endpoint_prefs(Call) of
         'true' -> check_inception(Call);
-        'false'  -> []
+        _  -> []
```

Comment:
I'd rather change `update_endpoint_prefs` and do `kapps_call:kvs_fetch(<<"use_endpoint_privacy">>, Call, 'false').` instead, and keep this case clause using `false`.

Reply:
Err, `(<<"use_endpoint_privacy">>, 'false', Call)`.